binding/binding_test.go

```diff
@@ -311,6 +318,15 @@ func TestBindingFormInvalidName2(t *testing.T) {
 		"map_foo=bar", "bar2=foo")
 }
 
+func TestBindingFormCollectionFormat(t *testing.T) {
+	testFormBindingForCollectionFormat(t, "POST",
+		"/?slice_multi=1&slice_multi=2&slice_csv=1,2&slice_ssv=1 2&slice_pipes=1|2", "/",
```

Comment:
Missing fuzzy tests here. Should try it with string type and see what happens if we feed pipes to csv param, csv to pipe param, csv to multi param etc…